Gemfile.lock

```diff
@@ -433,7 +433,7 @@ GEM
     loofah (2.8.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
-    lumberjack (1.2.4)
+    lumberjack (1.2.8)
```

Comment:
depency of guard, skipped manual check as we don't really use guard

Reply:
I thought @benhalpern uses `guard-rspec` quite heavily, but maybe I'm wrong/confused, both are known to happen :wink:

Reply:
maybe we can keep it around by forking it, I had to uncomment it in the [Rails 6.1 PR](https://github.com/forem/forem/pull/11333/files#diff-d09ea66f8227784ff4393d88a19836f321c915ae10031d16c93d67e6283ab55fR122-R123) because we need `rspec-rails 4.1` which is not supported by `guard-rspec`